src/core_modules/capture-core/components/FeedbackBar/FeedbackBar.component.js

```diff
@@ -17,110 +14,49 @@ const styles = () => ({
-            this.props.onClose();
-        }
-    }
-
-    getAction() {
-        const { feedback, classes } = this.props;
-
-        return (
-            <>
-                {
-                    (() => {
-                        if (!feedback.action) {
-                            return null;
-                        }
-
-                        return (
-                            <span
-                                className={classes.actionContainer}
-                            >
-                                {feedback.action}
-                            </span>
-                        );
-                    })()
-                }
-                <IconButton
-                    className={classes.closeButton}
-                    onClick={this.handleClose}
-                >
-                    <IconCross24 />
-                </IconButton>
-            </>
-        );
-    }
-
-    render() {
-        const { feedback } = this.props;
-        const { message, displayType } = feedback;
-        const isSnackBarOpen = isDefined(message) && !displayType;
-        const isDialogOpen = isDefined(message) && displayType === 'dialog';
-        return (
-            <React.Fragment>
-                <SnackBar
-                    open={isSnackBarOpen}
-                    anchorOrigin={Index.ANCHOR_ORIGIN}
-                    autoHideDuration={5000}
-                    onClose={this.handleClose}
-                    // $FlowFixMe[incompatible-type] automated comment
-                    message={<span>{message}</span>}
-                    action={this.getAction()}
-                />
-                {isDialogOpen && (
-                    <Modal
-                        hide={!isDialogOpen}
-                    >
-                        <ModalTitle>
-                            {
-                            // $FlowFixMe[prop-missing] automated comment
-                                isDialogOpen ? message && message.title : ''}
-                        </ModalTitle>
-                        <ModalContent>
-                            {
-                            // $FlowFixMe[prop-missing] automated comment
-                                isDialogOpen ? message && message.content : ''}
-                        </ModalContent>
-                        <ModalActions>
-                            <Button onClick={this.handleClose} primary>
-                                {i18n.t('Close')}
-                            </Button>
-                        </ModalActions>
-                    </Modal>
+    return (
+        <>
+            <AlertStack>
+                {isAlertBarOpen && (
+                    <AlertBar duration={5000}>
+                        {message}
+                    </AlertBar>
                 )}
-            </React.Fragment>
-        );
-    }
-}
-Index.displayName = 'FeedbackBar';
+            </AlertStack>
+            {isDialogOpen && (
+                <Modal hide={hide}>
```

Comment:
The modal hiding logic depends on the Redux store feedback and should not be handled internally by the FeedbackBar.component. Can this be [reverted](https://github.com/dhis2/capture-app/pull/3766/files#diff-e5c62664127a93fb774afd3dc3c03ece50718cd03217931b3d88530c015bc485L101) to `hide={!isDialogOpen}`?